component-dev-best-practice.md

```diff
@@ -40,18 +40,18 @@
     * [Standard action/trigger description](#standard-actiontrigger-description)
   * [Integration Application Definition of Done (IA-DoD) template](#integration-application-definition-of-done-ia-dod-template)
   * [Code library Definition of Done (CL-DoD) template](#code-library-definition-of-done-cl-dod-template)
-
-##Component development 
-###Component development prerequisites
+
+## Component development 
+### Component development prerequisites
  * Development team has all needed requirements
  * Access to the needed service with all needed permissions is granted
  * No open tasks that could block Component development process
  * The E.io’s GitHub repository must be checked if there is another implementation of the component under development. If it exists, new repository should not be created
  * Component architecture should be discussed and approved before start coding.
 
-###Definition of Done (CD-DoD)
```

Comment:
maybe `Component Development Definition of Done (CD-DoD)` instead of `Definition of Done (CD-DoD)`, so it's easier to find in document